LSP-json.sublime-settings

```diff
@@ -21,6 +21,7 @@
 			],
 		}
 	],
+	"auto_complete_selector": "punctuation.separator.mapping.key-value | (string - punctuation.definition.string.end)",
```

Comment:
Does `(string - punctuation.definition.string.end)` means that the autocomplete won't be triggered when the cursor is between quotes:
```
"key": "|"
```
?

Reply:
No it means it won't trigger when it is just outside of a string:

```
"key": ""|
        ^ this last double-quote has punctuation.definition.string.end
```

Reply:
The AC not triggering when you type the first `"` is because it isn't triggered when `insert_snippet` runs because of the automatic pairing of quotes.

Reply:
There's also a problem with PackageDev because it overrides the global `auto_complete_selector`, which unfortunately does not play well with vscode-json-language-server.